tests/test_emissions_tracker.py

```diff
@@ -76,7 +89,7 @@ def test_carbon_tracker_TWO_GPU_PRIVATE_INFRA_CANADA(
         self.assertEqual(
```

Comment:
The assertion just above breaks too (line 88, I cannot comment directly this line)